static/app/components/stream/group.tsx

```diff
@@ -49,7 +49,7 @@ import {
   Query,
 } from 'sentry/views/issueList/utils';
 
-const DiscoveryExclusionFields: string[] = [
+export const DiscoveryExclusionFields: string[] = [
```

Comment:
Should we move this into a constants file?

Also should it be called `DISCOVER_EXCLUSION_FIELDS` ?

Reply:
👍  moved and renamed!